docs/api.rst

```diff
@@ -1451,15 +1531,15 @@ For doing resource embedding, PostgREST infers the relationship between two tabl
 However, in cases where there's more than one foreign key between two tables, it's not possible to infer the relationship unambiguously
 by just specifying the tables names.
 
+For example, suppose you have the following ``orders`` and ``addresses`` tables.
+
+.. image:: _static/orders.png
+
 .. _target_disamb:
 
 Target Disambiguation
 ~~~~~~~~~~~~~~~~~~~~~
 
-For example, suppose you have the following ``orders`` and ``addresses`` tables:
-
-.. image:: _static/orders.png
-
 And you try to embed ``orders`` with ``addresses`` (this is the **target**):
```

Comment:
```suggestion
Let's try to embed ``orders`` with ``addresses`` (this is the **target**):
```

Or start with something different than "And", since it's not a continuation of the previous sentence anymore.

Reply:
Removed the previous change instead